Source/JavaScriptCore/bytecode/InlineCacheCompiler.cpp

```diff
@@ -1169,15 +1291,14 @@ void InlineCacheCompiler::generateWithGuard(AccessCase& accessCase, CCallHelpers
     JSGlobalObject* globalObject = m_globalObject;
     CCallHelpers& jit = *m_jit;
     JIT_COMMENT(jit, "Begin generateWithGuard");
-    StructureStubInfo& stubInfo = *m_stubInfo;
```

Comment:
Using `m_stubInfo` actually found dependent use of stubInfo information in code generation, so I think it is definitely better than using `stubInfo.` in random places (consistent use of `m_stubInfo`). Right now, asserting is very hard since it depends on DataIC / RepatchingIC, and making code super complicated.